static/app/views/issueDetails/updatedHeader.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import {GroupFixture} from 'sentry-fixture/group';
+import {OrganizationFixture} from 'sentry-fixture/organization';
+import {ProjectFixture} from 'sentry-fixture/project';
+import {TeamFixture} from 'sentry-fixture/team';
+import {UserFixture} from 'sentry-fixture/user';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import type {TeamParticipant, UserParticipant} from 'sentry/types';
+import {IssueCategory} from 'sentry/types';
+import StreamlinedGroupHeader from 'sentry/views/issueDetails/streamlinedHeader';
+import {ReprocessingStatus} from 'sentry/views/issueDetails/utils';
+
+describe('UpdatedGroupHeader', () => {
```

Comment:
Thanks for adding these tests!

One thing to remember here is that we are testing a lot of behavior in the old components like `AssignedTo` and the old sidebar/header. We will want to copy those tests over to this one or else we'll lose a lot of test coverage when we delete the old compoents.

Reply:
makes sense! ill do that in a separate PR, especially since im still working through some of the functionality of the header